MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1148 @@
+---
+layout: post
+title: Editors in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn about the supported editors in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Editors in .NET MAUI DataForm (SfDataForm)
+
+The data form supports several built-in editors as follows:
```

Comment:
The data form supports following built-in  editors to edit the data.

Reply:
content changed